public/css/team.css

```diff
@@ -76,9 +88,12 @@ p {
 	-moz-transition: all 0.5s ease-in-out;
 }
 
-.no-padding {
-	padding-left: 40;
-	padding-right: 40;
+/* Boxes */
+
+.team-container {
+	text-align: center;
+	padding-bottom: 40px;
+	background-color: var(--black);
 }
```

Comment:
Any particular reason for the 40px padding here? It just pushes away from the footer, leaving a weird gap